doc/api/cluster.md

```diff
@@ -661,6 +669,14 @@ Spawn a new worker process.
 
 This can only be called from the master process.
 
+```js
+  const cluster = require('cluster');
+
+  if(cluster.IsMaster){
+    cluster.fork();
+  }
+```
```

Comment:
This example should ideally do something with the spawned worker.

Reply:
Perhaps something like that:
```js
const http = require('http');
const cluster = require('cluster');
const numCPUs = require('os').cpus().length;

if (cluster.isMaster) {
  for (let i = 0; i < numCPUs; i++) {
    // Workers will be created depending on the CPUs numbers.
    const worker = cluster.fork();

    // Handle messages from the worker.
    worker.on('message', (message) => {
      console.log(`Master received message from worker ${worker.id}: ${message}`);
    });

    // Simulate a task for the worker (e.g., sending a message).
    setTimeout(() => {
      worker.send('Hello Worker!');
    }, 2000);
  }
} else {
  // Worker process logic.
  http.createServer((req, res) => {
    res.writeHead(200);
    res.end('hello world');

    // Send a message to the master.
    process.send(`Hello Master from worker ${cluster.worker.id}!`);
  }).listen(3000);
}
```